diff --git a/lib/src/utils/get-sorted-nodes.js b/lib/src/utils/get-sorted-nodes.js
index 7a1560901bc3e238bc745121f94a5d6cc17451db..f2e67dd8a97ef31743d7b0d5ec46a59b8b811bf2 100644
--- a/lib/src/utils/get-sorted-nodes.js
+++ b/lib/src/utils/get-sorted-nodes.js
@@ -51,6 +51,10 @@ var getSortedNodes = function (nodes, options) {
     }
     for (var _a = 0, importOrder_1 = importOrder; _a < importOrder_1.length; _a++) {
         var group = importOrder_1[_a];
+        if (group === '<BLANK_LINE>') {
+            finalNodes.push(constants_1.newLineNode);
+            continue;
+        }
         var groupNodes = importOrderGroups[group];
         if (groupNodes.length === 0)
             continue;